spec/features/homepage_spec.rb

```diff
@@ -23,7 +23,9 @@
       # Expect the system create of work2 to be later than that of work1.
       # This helps verify that 'recently uploaded' is looking at
       # 'date_uploaded_dtsi'  and not 'system_create_dtsi'.
-      expect(DateTime.parse(work2.to_solr['system_create_dtsi']).getlocal).to be >= DateTime.parse(work1.to_solr['system_create_dtsi']).getlocal
+      document1 = SolrDocument.new(GenericWorkIndexer.new(resource: work1).to_solr)
+      document2 = SolrDocument.new(GenericWorkIndexer.new(resource: work2).to_solr)
+      expect(document2['system_create_dtsi']).to be >= document1['system_create_dtsi']
```

Comment:
Since the work model is changed to Monograph it seems strange to use the GenericWorkIndexer. I think this should work, or perhaps the solr doc could be fetched from solr instead of rebuilt?
```suggestion
      document1 = SolrDocument.new(Hyrax::Indexers::ResourceIndexer.for(resource: work1).to_solr)
      document2 = SolrDocument.new(Hyrax::Indexers::ResourceIndexer.for(resource: work2).to_solr)
      expect(document2['system_create_dtsi']).to be >= document1['system_create_dtsi']
```

Reply:
I updated the PR to use `document1 = Hyrax.index_adapter.connection.get("select", params: { q: "id:#{work1.id}" })["response"]["docs"].first` to retrieve the Solr data of the works. Let me know if you have a recommended approach.